usvm-ts/src/test/kotlin/org/usvm/util/TsTestResolver.kt

```diff
@@ -241,7 +253,7 @@ open class TsTestStateResolver(
                 val value = finalStateMemory.read(lValue)
                 val ref = model.eval(value)
                 // TODO mistake with signature, use TypeStream instead
-                resolveExpr(ref, EtsClassType(ctx.scene.projectAndSdkClasses.first().signature))
+                resolveExpr(ref, value, EtsClassType(ctx.scene.projectAndSdkClasses.first().signature))
```

Comment:
What? Fake objects have "signature" = "signature of some random class from scene"?